packages/trace-viewer/src/ui/networkResourceDetails.tsx

```diff
@@ -77,8 +77,12 @@ const RequestTab: React.FunctionComponent<{
   }, [resource]);
 
   return <div className='network-request-details-tab'>
-    <div className='network-request-details-header'>URL</div>
-    <div className='network-request-details-url'>{resource.request.url}</div>
+    <div className='network-request-details-header'>General</div>
+    <div className='network-request-details-url'>{`URL: ${resource.request.url}`}</div>
+    <div className='network-request-details-general'>{`Method: ${resource.request.method}`}</div>
+    <div className='network-request-details-general'>{`Status Code: ${resource.response.status >= 200 && resource.response.status < 400
```

Comment:
Let's also show `resourse.response.statusText`? Something like `Status Code: 200 OK`.

Reply:
I've added the `resourse.response.statusText` @dgozman , @yury-s